src/common/github-flavored-markdown.js

```diff
@@ -4,6 +4,12 @@ import { escapeHtml } from '../utils/string-utils.js';
 const audioUrlPattern = new RegExp(`\\.(${['aac', 'm4a', 'mp3', 'oga', 'ogg', 'wav'].join('|')})$`, 'i');
 const videoUrlPattern = new RegExp(`\\.(${['mp4', 'm4v', 'ogv', 'webm', 'mpg', 'mpeg'].join('|')})$`, 'i');
 
+// Matches both URLs in e.g.: [![alt](cdn://image.png "image title")](cdn://some-target)
+const imageInsideOfHyperlinkPattern = /(\[!\[[^\]]*\]\()(\S*?)((?:\s+[^)]*)?\s*\)]\()(\S*?)((?:\s+[^)]*)?\s*\))(?!\])/g;
```

Comment:
Jeeeesuuuus....

Reply:
I'm so sorry, but I cannot think of anything better to do the job.